src/Bridge/Symfony/DependencyInjection/SonataFormExtension.php

```diff
@@ -51,5 +53,17 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader->load('validator.xml');
 
         $container->setParameter('sonata.form.form_type', $config['form_type']);
+
+        $this->configureSerializerFormats($config);
+    }
+
+    /**
+     * @param mixed[] $config
+     */
+    public function configureSerializerFormats($config): void
```

Comment:
Why this method needs to be public?

Reply:
I move it form CoreBundle and didn't check.